packages/runtime/src/runtimes/react/controls/rich-text-v2/EditableTextV2/editable-text-v2.tsx

```diff
@@ -77,7 +79,57 @@ export function EditableTextV2({ text, control }: Props) {
   }, [editor, control])
 
   const renderElement = useCallback(({ attributes, children, element }: RenderElementProps) => {
+    const blockStyles = [useStyle({ margin: 0 })]
```

Comment:
This is temporary. When I work on composable render functionality this would be moved into the block plugin

Reply:
Cool. Do you want to add a TODO comment?